### ISSUE TITLE
Library referring to another Library

### ISSUE THREAD

**Issue by lincolnminto** [mulesoft-labs/raml-for-jax-rs#309]
When I use multiple Libraries references it is appended to the java entity reference the defined alias from the RAML library type name causing a _Compilation failure_.

**RAML Library:**
```raml
#%RAML 1.0 Library
uses:
  lib2: lib2.raml
  ...
  properties:
    list:
        type: array
        items: lib2.File2
```

**JAVA POJO:**
```java
List<Lib2File2> getList();
```

**Exception:**
```makeFile
cannot find symbol
[ERROR] symbol:   class Lib2File2
```

Please, check the attached files.

[test-multiple-raml-libraries.zip](https://github.com/mulesoft-labs/raml-for-jax-rs/files/1388525/test-multiple-raml-libraries.zip)

**Comment by jpbelang** [comment]
This is interesting.  I'm not sure I like the generated name.  I think I'd prefer a separate package.

I'm currently finishing something on another project.  I should be back on this project next week.  Thanks for the example.

**Comment by lincolnminto** [comment]
This behavior happens whenever you reference to a library. Even if you are referencing it from the RAML root definition.

```raml
#%RAML 1.0
title: My Root Element
version: 1.0.0
...
uses:
  lib: library.raml

/Commit:
      post:
       displayName: Commit
       description: By passing in the appropriate options, you can commit
       body:
         application/json:
               # formParameters:
           schema: lib.MyType
        responses:
          '201':
            body:
              application/json:
                  schema: lib.MyType
```

It generates the exception:

```makeFile
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /service/MyService.java
:[18,13] cannot find symbol
  symbol:   class LibMyType
  location: package model
```

**Comment by jpbelang** [comment]
Fixed on release/2.2.0 (through the raml-to-pojo library).
